testssl.sh

```diff
@@ -3933,11 +3933,11 @@ sclient_connect_successful() {
 determine_tls_extensions() {
      local proto addcmd
      local success
-     local alpn=""
+     local alpnParam=""
      local savedir
      local nrsaved
 
-     "$HAS_ALPN" && alpn="h2-14,h2-15,h2"
+     "$HAS_ALPN" && alpnParam="-nextprotoneg h2-14,h2-15,h2"
```

Comment:
$HAS_ALPN" && alpnParam="-nextprotoneg h2-14,h2-15,h2".

It should either read  ~ ``$HAS_NPN" && npnParam="-nextprotoneg h2-14,h2-15,h2"`` or ~ ``$HAS_ALPN" && alpnParam="-alpn h2-14,h2-15,h2".``